presidio-tester/cmd/presidio-tester/testdata/ocr-result.json

```diff
@@ -0,0 +1 @@
n":677,"endPosition":679},{"xLocation":73,"width":225,"yLocation":800,"height":831,"text":"Conduct.","startPosition":680,"endPosition":688},{"xLocation":240,"width":293,"yLocation":802,"height":831,"text":"For","startPosition":689,"endPosition":692},{"xLocation":306,"width":392,"yLocation":809,"height":831,"text":"more","startPosition":693,"endPosition":697},{"xLocation":406,"width":602,"yLocation":800,"height":831,"text":"information","startPosition":698,"endPosition":709},{"xLocation":616,"width":674,"yLocation":809,"height":831,"text":"see","startPosition":710,"endPosition":713},{"xLocation":685,"width":739,"yLocation":800,"height":831,"text":"the","startPosition":714,"endPosition":717},{"xLocation":751,"width":839,"yLocation":800,"height":831,"text":"Code","startPosition":718,"endPosition":722},{"xLocation":851,"width":886,"yLocation":800,"height":831,"text":"of","startPosition":723,"endPosition":725},{"xLocation":897,"width":1040,"yLocation":800,"height":831,"text":"Conduct","startPosition":726,"endPosition":733},{"xLocation":29,"width":94,"yLocation":858,"height":891,"text":"FAQ","startPosition":734,"endPosition":737},{"xLocation":107,"width":141,"yLocation":865,"height":887,"text":"or","startPosition":738,"endPosition":740},{"xLocation":152,"width":281,"yLocation":860,"height":887,"text":"contact","startPosition":741,"endPosition":748},{"xLocation":27,"width":485,"yLocation":912,"height":951,"text":"opencode@microsoft.com","startPosition":749,"endPosition":771},{"xLocation":498,"width":568,"yLocation":912,"height":943,"text":"with","startPosition":772,"endPosition":776},{"xLocation":582,"width":640,"yLocation":921,"height":951,"text":"any","startPosition":777,"endPosition":780},{"xLocation":652,"width":819,"yLocation":912,"height":943,"text":"additional","startPosition":781,"endPosition":791},{"xLocation":833,"width":1000,"yLocation":914,"height":950,"text":"questions","startPosition":792,"endPosition":801},{"xLocation":1013,"width":1047,"yLocation":921,"height":943,"text":"or","startPosition":802,"endPosition":804},{"xLocation":27,"width":219,"yLocation":971,"height":998,"text":"comments.","startPosition":805,"endPosition":814}],"text":"This project is created by David Johnson and welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com or contact (212) 555-1234. When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA| This project has adopted the Microsoft Open Source Code of Conduct. For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments. "}
```

Comment:
Are these taken from the tests folder? moved or copied?

Reply:
Moved from the tests folder.
Tests are now referencing to the new location.